openslides_backend/action/default_schema.py

```diff
@@ -13,20 +13,22 @@ class DefaultSchema:
     def __init__(self, model: Model) -> None:
         self.model = model
 
-    def get_create_schema(
-        self, properties: Iterable[str], required_properties: Iterable[str]
+    def get_default_schema(
+        self,
+        required_properties: Iterable[str] = [],
+        optional_properties: Iterable[str] = [],
     ) -> Dict:
         """
         Returns a default create schema with properties and required properties as given.
         """
         return {
             "$schema": schema_version,
```

Comment:
Please readd title and description for create, update and delete schemas. For default schema it can be setup by optional parameters as you do for the single item schema.